packages/jest-circus/src/legacy_code_todo_rewrite/jest-expect.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+import type {RawMatcherFn} from 'types/Matchers';
+
+const expect = require('jest-matchers');
+
+const {
+  addSerializer,
+  toMatchSnapshot,
+  toThrowErrorMatchingSnapshot,
+} = require('jest-snapshot');
+
+type JasmineMatcher = {
+  (): JasmineMatcher,
+  compare: () => RawMatcherFn,
+  negativeCompare: () => RawMatcherFn,
+};
+
+module.exports = (config: {expand: boolean}) => {
+  global.expect = expect;
+  expect.setState({
+    expand: config.expand,
+  });
+  expect.extend({toMatchSnapshot, toThrowErrorMatchingSnapshot});
+  (expect: Object).addSnapshotSerializer = addSerializer;
```

Comment:
We should add the `addSnapshotSerializer` type to Expect so we don't need to cast here.